upi/vsphere/lb/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "lb_ip_address" {
+  type = string
+}
+variable "bootstrap_ip_address" {
+  type = string
+}
+
+variable "control_plane_ip_addresses" {
+  type = list(string)
+}
+
+variable "compute_ip_addresses" {
+  type = list(string)
+}
+
+variable "ssh_public_key_path" {
+  type    = string
+  default = "~/.ssh/openshift-dev.pub"
```

Comment:
i wouldn't want to keep this default in one of the modules.. the main variable could certainly have the default.